tests/integration/components/ui-message-test.js

```diff
@@ -0,0 +1,317 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('ui-message', 'Integration | Component | ui-message', {
+  integration: true,
+});
+
+test('it renders properly', function(assert) {
+  assert.expect(2);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check wrapper <div>.
+  assert.strictEqual($component.hasClass('ui'), true, 'Component\'s wrapper has \'ui\' css-class');
+  assert.strictEqual($component.hasClass('message'), true, 'Component\'s wrapper has \' message\' css-class');
+});
+
+test('size renders properly', function(assert) {
+  assert.expect(8);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check component's additional CSS-classes.
+  let additioanlCssClasses = 'small large huge massive';
+  this.set('class', additioanlCssClasses);
+
+  Ember.A(additioanlCssClasses.split(' ')).forEach((cssClassName, index) => {
+    assert.strictEqual(
+    $component.hasClass(cssClassName),
+    true,
+    'Component\'s wrapper has additional css class \'' + cssClassName + '\'');
+  });
+
+  this.set('class', '');
+  Ember.A(additioanlCssClasses.split(' ')).forEach((cssClassName, index) => {
+    assert.strictEqual(
+    $component.hasClass(cssClassName),
+    false,
+    'Component\'s wrapper hasn\'t additional css class \'' + cssClassName + '\'');
+  });
+});
+
+test('type renders properly', function(assert) {
+  assert.expect(12);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check component's additional CSS-classes.
+  let additioanlCssClasses = 'warning info positive success negative error';
+  this.set('class', additioanlCssClasses);
```

Comment:
Test for 'type' must use conponent's 'type' property of component instead of common property 'class'.
Like that:
`{{ui-message type="info"}}` & assert($component.hasClass('info'), true, ...)